c/pthread-lit/fkp2014_true-unreach-call.i

```diff
@@ -654,25 +654,26 @@ void __VERIFIER_assert(int cond) {
   }
   return;
 }
-void __VERIFIER_atomic_assert(int cond) {
```

Comment:
No, all is good. I thought that `__VERIFIER_assert` was not defined, but now I looked more carefully and I saw that it is there. So this should be fine.